config-production.yaml

```diff
@@ -1,8 +1,10 @@
 # Production configuration for the cf-plugin-backup pipelines
 # Note that the secrets are not here; they are in $CONCOURSE_SECRETS_FILE
 
-src-name: SUSE/cf-plugin-backup
-src-repo: git@github.com:SUSE/cf-plugin-backup.git
+src-owner:                   SUSE
+src-repo:                         cf-plugin-backup
+src-name:                    SUSE/cf-plugin-backup
+src-repo-ref: git@github.com:SUSE/cf-plugin-backup.git
```

Comment:
Bike shedding: `src-repo-url`?
(Feel free to ignore this comment if you don't like it)

Reply:
Heh. IIRC I used that first and then went to `...-ref`.

On the topic of the test steps ... My thinking was that when the tag is set the commit already went through the regular master and as such is safe ... OTOH you are quite right, somebody might make a mistake and set the tag on a bad commit. So, the tests should be run.

It should be good enough to run them as part of the task, and not as separate tasks, right ?

So, will do on the test steps, and ignoring the bikeshed

Reply:
Bikeshed ignoring is fine.

I was thinking that we _should_ just reuse the steps, and duplicate the pipeline instead (`cf-plugin-backup-release`), maybe? So the exact release binaries can live in a bucket too, in case we need to do something with that later…

Reply:
Oh. I see. Note that currently the binaries only go into the release, and not into any bucket.

I guess in a duplicated pipeline I could attach the s3 uploads to the release task as well.
Maybe create a single tarball from the meta data and the assets, and upload that, into the existing bucket, with a proper name.

I am warming up to the idea.

Reply:
Ok. Commit 755bf74e70ba2866af946d5aa0e18c9709ce1680
addresses most of the comments. It is however still the task in master pipeline.

I will now twin it off and make the other necessary changes.
Twinned off I can also change `src-tags` to be `src` for that pipeline, and drop the unfiltered `src`.
No special mappings.